src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator.js';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <Calculator />
```

Comment:
**You have done great job!**

However, in your app we have some places to improve and fix. For starting, all the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Kindly delete them :)

<img width="416" alt="Screenshot 2023-05-31 at 03 55 42" src="https://github.com/ahmed025566/Math-Magicians/assets/69011963/272c1a10-2435-4646-976d-44d320da08dc">